server/views/components/wip/dateField.njk

```diff
@@ -0,0 +1,18 @@
+{% from "moj/components/date-picker/macro.njk" import mojDatePicker %}
+
+{% macro renderDateField(fieldName, field, options, values, errors) %}
+
+  {{ mojDatePicker({
+    name: fieldName,
+    id: fieldName,
+    label: {
+      text: field.label,
+      classes: "govuk-label--m"
+    },
+    hint: { text: field.hint } if field.hint else undefined,
+    leadingZeros: "true",
```

Comment:
I know we kinda had this conversation before but I still see this error when `leadingZeros` is a string:

```
Uncaught TypeError: this.config.leadingZeros.toLowerCase is not a function
```

And as a result the fancy date picker doesn't show (just the text input)

Reply:
it _totally_ does not work in _this_ template if u make it a boolean

Reply:
So, as discussed offline, I was getting this error not sure why...a fresh `nvm use && rm -rf node_modules && npm install` somehow solved the problem (even tho I'm pretty sure I've run an npm install _**very recently**_)...so perhaps npm using an older version somehow?

Odd but at least this works (although my preference would be to rather have this as a boolean _**if it worked**_ 😬 )